sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
+	"Resources": {
+		"ServiceRoot": {
+			"PropertyRequirements": {
+				"Chassis": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ChassisCollection"]
+				},
+				"Managers": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ManagerCollection"]
+				},
+				"SessionService": {
```

Comment:
Ditto re: `ReadRequirement`.

Reply:
I think we can remove SessionService altogether; SessionService is mandatory in Redfish.

Reply:
@mraineri, I cannot find where ``SessionService`` is specified as a mandatory property of the Service Root. I looked at the [ServiceRoot.v1_8_0](https://redfish.dmtf.org/schemas/v1/ServiceRoot.v1_8_0.yaml) schema. Please let me know where to look.

Following up on our conversation this past Friday morning, it appears sushy treats it as optional when [``class SessionOrBasicAuth``](https://opendev.org/openstack/sushy/src/commit/619445ce5446c6bdcec4776823a76f895d18460c/sushy/auth.py#L210-L228) authentication is used. An exception is raised by the call on [line 218](https://opendev.org/openstack/sushy/src/commit/619445ce5446c6bdcec4776823a76f895d18460c/sushy/auth.py#L218) when the ``SessionService`` property is not present. Notice that it falls back to basic authentication.

Reply:
As per our discussion today evening, SessionService is optional in Redfish hence kept in the profile definition. However removed the comparison and value for the same.